<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPlaying</class>
 <widget class="QDialog" name="DlgPlaying">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>703</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>871</width>
    <height>870</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>拼图小游戏</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QToolButton
{
	min-height: 25px;
	min-width: 70px;
	max-height: 25px;
	max-width: 70px
}</string>
  </property>
  <widget class="QLCDNumber" name="lcdStep">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>65</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnreturn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 20pt &quot;方正蕴黑 简 ExtraBold&quot;;
background-color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>386</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 36, 36);font: 800 20pt &quot;方正蕴黑 简 ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>还原</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>458</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 36, 36);font: 800 20pt &quot;方正蕴黑 简 ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>提示</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 35pt &quot;方正超值体 简 ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>步数：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="time_labl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 35pt &quot;方正超值体 简 ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>耗时：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::PlainText</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdStep_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>65</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRestart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 36, 36);font: 800 20pt &quot;方正蕴黑 简 ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>重开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>530</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>530</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>529</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 20pt &quot;方正蕴黑 简 ExtraBold&quot;;
background-color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
